qml/pages/CoverArtList.qml

```diff
@@ -113,9 +115,9 @@ BackgroundItem
 
     Label {
         id: vtext
-        anchors.verticalCenter: coverImg.status!=Image.Error && textvisible? undefined : parent.verticalCenter
-        anchors.bottom: coverImg.status!=Image.Error && textvisible? parent.bottom : undefined
-        anchors.bottomMargin: coverImg.status!=Image.Error && textvisible? Theme.paddingSmall : undefined
+        y: coverImg.status!=Image.Error && textvisible
```

Comment:
Is the single `y` at the start of this line (118) correct?
Just wondering, a {Yes|No} reply is fully sufficient.

Reply:
This changing is fixing a label placement issue:
* go to the artist list and select one artist without image (notice that the label is centred in the square),
* pull down and search for an artist image representation, select one,
* you're sent back to the artist page, and the artist has now an image representation,
* go back to the artist list page and notice that the artist has also an image now. But the label is still centred.

The original code looks good to me, and I was almost sure I've already seen code like that, where the vertical centre is set under some conditions, otherwise undefined ; while a bottom anchor is set for the negation of such conditions. But it's not working here and after some hours trying to figure out why, I gave up and used absolute positioning via the `x` or `y` properties. It's less intentional, but well, it's working.

By the way, yes, `y` is a property for an item, it's the vertical position of its top border from the top of its parent (positive going down). I've checked that the algebra here is working both in portrait and landscape.